library/alloc/src/sync.rs

```diff
@@ -485,6 +485,20 @@ impl<T> Arc<T> {
     ///
     /// This will succeed even if there are outstanding weak references.
     ///
+    // FIXME: when `Arc::into_inner` is stabilized, add this paragraph:
+    /*
+    /// It is strongly recommended to use [`Arc::into_inner`] instead if you don't
+    /// want to keep the `Arc` in the [`Err`] case.
+    /// Immediately dropping the [`Err`] payload, like in the expression
+    /// `Arc::try_unwrap(this).ok()`, can still cause the strong count to
```

Comment:
Talking for the future here, but if/when this feature gets stabilized, it will be nice to have a (clippy?) lint for this 🙂

Reply:
I don't know if this has been done before, but adding a `FIXME` comment about opening an issue for it on clippy would ensure this is not forgotten when this becomes stable.